packages/www/pages/dashboard/streams/[id]/index.tsx

```diff
@@ -343,17 +345,26 @@ const ID = () => {
                   <DropdownMenuGroup>
                     <Record
                       stream={stream}
-                      revalidate={revalidate}
+                      revalidate={queryClient.invalidateQueries()}
```

Comment:
We want to pass the `invalidateQueries` method down, but _not_ execute it...

This is executing `invalidateQueries` and passing down whatever it gets in return.

Reply:
Also, same as comments above: we don't want to invalidate ALL the queries.